src/main/webapp/timestamps.js

```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {String} a time range from
+ * `startTimestampMs` to `endTimestampMs`.
+ */
```

Comment:
This function is patched from pull request #228 . Please ignore this function for this pull request. Thank you!

Reply:
Will you remove this before merging this PR?

Reply:
Yes :D